Gemfile

```diff
@@ -86,9 +86,11 @@ gem 'devise-guests', '~> 0.6'
 
 gem 'config'
 
+gem 'github_api', github: 'piotrmurach/github'
```

Comment:
Why do we need this from github? Maybe leave a comment so we know when to move back to a release?